src/main/scala/org/camunda/feel/impl/parser/FeelParser.scala

```diff
@@ -118,7 +119,29 @@ object FeelParser {
 
   // 2 a)
   private def textualExpression[_: P]: P[Exp] =
-    P(functionDefinition | forExpression | ifExpression | quantifiedExpression | expression2)
+    P(expressionInBrackets | functionDefinition | forExpression | ifExpression | quantifiedExpression | expression2)
+
+  private def expressionInBrackets[_: P]: P[Exp] =
```

Comment:
I think the word brackets is generally used specifically for square brackets `[ ]`. Instead, the word parentheses is used for round brackets `( )`. See [bracket](https://en.wikipedia.org/wiki/Bracket).